bot.py

```diff
@@ -358,9 +366,11 @@ async def add_summoner(ctx, *, message):
         await ctx.send(embed=create_embed(embed_data))
 
         # display list of summoners
+        # TODO this shouldn't call another decorator function.
         await display_current_list_of_summoners(ctx)
```

Comment:
@HelloFive @jjaykim @jung1411 
Idk who worked on this part - but we shouldn't call another decorator function from another decoratior function. 

Nothing wrong in terms of code itself - but we should create new `create_embed_list` function that creates embed data and this function should return the embed and end.